Dockerfile

```diff
@@ -19,4 +17,12 @@ FROM nginxinc/nginx-unprivileged AS run
 
 COPY default.conf /etc/nginx/conf.d/default.conf
 COPY --from=build /app/build /usr/share/nginx/html
+
+WORKDIR /usr/share/nginx/html
+COPY .env scripts/env.sh ./
+
+# Run script which initializes env vars to fs
+RUN chmod +x env.sh
+RUN ./env.sh
+
```

Comment:
Hier zit nog een probleem @violetina, chmod is niet toegestaan in `nginx-unprivileged`. Als ik het weglaat runt het script wel, maar slaagt het er niet in om weg te schrijven naar `env-config.js`

(details:)
```
rm: cannot remove './env-config.js': Permission denied
touch: cannot touch './env-config.js': Permission denied
./env.sh: line 8: ./env-config.js: Permission denied
./env.sh: line 26: ./env-config.js: Permission denied
./env.sh: line 29: ./env-config.js: Permission denied
```

Weet jij hoe we dit kunnen oplossen?